pkg/cmd/plan.go

```diff
@@ -200,6 +201,12 @@ func importCommand(c *cli.Context) error {
 	}
 	dstPath := filepath.Join(cfg.Dirs().Home(), "plans", baseDest)
 
+	// check if path exists
```

Comment:
I don't want to have to add `|| true` to every `testground plan import` command we ever write (currently in `make test` and in `make test-integration`).

We also don't want to `rm -rf` anything outside of the working directory and that includes $TESTGROUND_HOME.

Therefore I suggest we don't fail when a path exists for a given testplan, but just display a warning.

---

Tests would still fail if users have modified sources at $TESTGROUND_HOME, but in the general case that will work. WDYT?

Reply:
Yeah, seems okay!